config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml

```diff
@@ -208,6 +208,60 @@ periodics:
   - 'perfDashPrefix: gce-100Nodes-1.16'
   - 'perfDashJobType: performance'
   - 'perfDashBuildsCount: 500'
+- annotations:
+    fork-per-release-cron: 0 15 * * *, 0 21 * * *
+    fork-per-release-generic-suffix: "true"
+    testgrid-dashboards: sig-scalability-kubemark, sig-release-job-config-errors
+    testgrid-num-failures-to-alert: "1"
+    testgrid-tab-name: kubemark-1.16-500
+  cron: 0 9 * * *
+  labels:
+    preset-dind-enabled: "true"
+    preset-e2e-kubemark-common: "true"
+    preset-k8s-ssh: "true"
+    preset-service-account: "true"
+  name: ci-kubernetes-kubemark-500-gce-stable1
+  spec:
+    containers:
+    - args:
+      - --repo=k8s.io/kubernetes=release-1.16
+      - --repo=k8s.io/perf-tests=release-1.16
+      - --root=/go/src
+      - --timeout=140
+      - --scenario=kubernetes_e2e
+      - --
+      - --cluster=kubemark-500
+      - --extract=ci/latest
+      - --gcp-master-size=n1-standard-4
+      - --gcp-node-image=gci
+      - --gcp-node-size=n1-standard-8
+      - --gcp-nodes=8
+      - --gcp-project-type=scalability-project
+      - --gcp-zone=us-east1-b
+      - --kubemark
+      - --kubemark-nodes=500
+      - --provider=gce
+      - --test=false
+      - --test_args=--ginkgo.focus=xxxx
+      - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
+      - --test-cmd-args=cluster-loader2
+      - --test-cmd-args=--nodes=500
+      - --test-cmd-args=--provider=kubemark
+      - --test-cmd-args=--report-dir=/workspace/_artifacts
+      - --test-cmd-args=--testconfig=testing/density/config.yaml
+      - --test-cmd-args=--testconfig=testing/load/config.yaml
+      - --test-cmd-args=--testoverrides=./testing/load/kubemark/500_nodes/override.yaml
```

Comment:
Don't we need to enable prometheus at least?

Reply:
It is not enabled in gce-100 job:
https://github.com/kubernetes/test-infra/blob/21e6dec645d0ed5f42d44b32ef62043555d1b026/config/jobs/kubernetes/sig-release/release-branch-jobs/1.16.yaml#L199
should we enable it in both places?

Reply:
Similarly use_simple_latency_query.yaml - should we add that in both jobs?

Reply:
Probably...

But given it's not there, let's clean it up separately.